sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -1445,6 +1440,40 @@ SIREPO.app.directive('simplePanel', function(appState, panelState) {
     };
 });
 
+SIREPO.app.directive('simulationStoppedStatus', function(authState) {
+    return {
+        restrict: 'A',
+        scope: {
+            simState: '=simulationStoppedStatus',
+        },
+        template: [
+            '<div class="col-sm-12" ng-bind-html="message()"><br><br></div>',
+        ].join(''),
+        controller: function($scope, $sce) {
+            $scope.message = function() {
+                if ($scope.simState.isStatePurged()) {
+                    return $sce.trustAsHtml([
+                        '<div>Simulation data purged on ' + $scope.simState.getDbUpdateTime() + ' UTC.</div>',
+                        '<div>Upgrade to <a href="https://radiasoft.net/sirepo" target="_blank">Sirepo ' + authState.premiumOrEnterprise() + '</a> for persistent data storage.</div>',
```

Comment:
See previous comment about redirects to pay.

Reply:
https://github.com/radiasoft/sirepo/pull/2690/commits/68d5a2d040d51f8ae5ba41517c9b0571c45a1f72